post-processing/post_processing_tutorial.md

```diff
@@ -0,0 +1,25 @@
+## Post-Processing Tutorial
+
+### Overview
+
+The post-processing scripts provided with the ExCALIBUR tests package are intended to grant users a quick starting point for visualising benchmark results with basic graphs and tables.
+
+There are three main post-processing components:
+- **`Perflog parsing:`**
+  - Data from benchmark performance logs are stored in a pandas DataFrame.
+- **`Data filtering:`**
+  - If more than one perflog is used for plotting, DataFrames from individual perflogs are concatenated together into one DataFrame.
+  - The DataFrame is then filtered, keeping only relevant rows and columns.
+- **`Plotting:`**
+  - A filtered DataFrame is passed to a plotting script, which produces a graph and embeds it in a simple HTML file. (`TODO: Embed graphs in GitHub Pages.`)
+  - Users may run the plotting script to generate a generic bar chart. Graph settings should be specified in a configuration YAML file. (`TODO: Add scaling and regression plots.`)
```

Comment:
Should to-dos be kept or removed from the tutorial? It seems pertinent to inform users of missing features and this is easier than reworking more of the tutorial later, but it would look a bit neater to not have them there.

Reply:
I'll put them in a separate section at the end. I'll also re-name this into a README, and pick and choose for the actual tutorial. But it's very close! :)